napari/layers/shapes/shapes.py

```diff
@@ -1195,6 +1231,34 @@ def _get_state(self):
         )
         return state
 
+    @property
+    def _indices_view(self):
+        return np.where(self._data_view._displayed)[0]
+
+    @property
+    def _view_text(self) -> np.ndarray:
+        """Get the values of the text elements in view
+
+        Returns
+       -------
+       text : (N x 1) np.ndarray
+           Array of text strings for the N text elements in view
```

Comment:
```suggestion
        -------
        text : (N x 1) np.ndarray
            Array of text strings for the N text elements in view
```

How did these even happen so consistently??? 😂

Reply:
I have no idea! I just press tab!